DEVELOPING.md

```diff
@@ -0,0 +1,44 @@
+# Configuration Parameter Contributions
+
+This is a brief guide for developers interested in making open-source contributions to the configuration parameters for 
+java-manta. [Java-Manta]((http://joyent.github.com/java-manta)) is a community-maintained Java SDK for interacting with 
+Joyent's Manta object storage system.
+
+### Recommended Prerequisite References
+- [Using The SDK](https://github.com/joyent/java-manta#usage).
+- [Contributing To Java-Manta](https://github.com/joyent/java-manta/blob/master/CONTRIBUTING.md).
+- [Configuration Documentation](https://github.com/joyent/java-manta/blob/master/USAGE.md#configuration).
+- [General Examples](https://github.com/joyent/java-manta#general-examples).
+ 
+### Adding a Configuration Parameter to the SDK
+
+Configuration parameters allow users to flexibly leverage SDK features to match their performance, authentication, 
+and encryption requirements.
+
+To submit a new Configuration Parameter to the Java-Manta SDK, do the following:
```

Comment:
Perhaps it would be more accurate to say: "To add a new...". Also, why is "Configuration Parameter" capitalized?

Reply:
Changed.